content/client-server-api/modules/event_replacements.md

```diff
@@ -199,10 +192,11 @@ Note that there can be multiple events with an `m.replace` relationship to a
 given event (for example, if an event is edited multiple times). These should
 be [aggregated](#aggregations) by the homeserver.
 
-The aggregation format of `m.replace` relationships gives the `event_id`,
-`origin_server_ts`, and `sender` of the **most recent** replacement event. The
-most recent event is determined by comparing `origin_server_ts`; if two or more
-replacement events have identical `origin_server_ts`, the event with the
+The aggregation format of `m.replace` relationships gives the **most recent**
+replacement event, formatted as normal for the client-server API (see [Room Event Format](#room-event-format)).
```

Comment:
```suggestion
replacement event. {{< changed-in v="1.7" >}} That event is formatted as normal for the client-server API (see [Room Event Format](#room-event-format)).
```

arguably we can also use less words because we're already in the CS API:
```suggestion
replacement event. {{< changed-in v="1.7" >}} That event is formatted [as normal](#room-event-format) for clients.
```

Reply:
Now that I've moved the `changed-in` to the top of the section I don't think this extra changed-in is helpful. I've applied the wording changes, with tweaks.